Week1/homework/app.js

```diff
@@ -1,11 +1,140 @@
 'use strict';
 {
+  //2.1
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'brain_camp ',
+    'sophie_washington_secret_santa',
+    'el_deafo',
+    'american_born_chinese',
+    'queen_of_the_world!',
```

Comment:
Do not use special characters because then you can't use these names as element IDs. Make sure these names conform to the rules for JavaScript variable names: letters, digits and the underscore.